resources/views/createCourse.blade.php

```diff
@@ -0,0 +1,154 @@
+
+@extends('layouts.app')
+
+@section('title', 'Register a course')
+
+@section('content')
+<div class="padded content">
+	<h1>Register a course</h1>
+	<form class="ui error form" method="POST" action="/courses">
+		@csrf
+		<!-- Course name -->
+		<div class="field">
+			<label for="courseName">Course name</label>
+			<input type="text" name="courseName" id="courseName" required>
+		</div>
+		<!-- Teacher(s) -->
+		<div class="field" title="Are you giving this course with another teacher?">
+			<label for="teachers">Teacher(s)</label>
+			<select class="ui fluid search dropdown" name="teachers" id="teachers" multiple required>
+				<option value="1">Mr. Monday</option>
+				<option value="2">Mrs. Tuesday</option>
+				<option value="3">Ms. Wednesday</option>
+				<option value="4">Mr. Thursday</option>
+				<option value="5">Mrs. Friday</option>
+			</select>
+		</div>
+		<!-- Course semester -->
+		<div class="field">
+			<label for="courseSemester">Course semester</label>
+			<input type="text" name="courseSemester" id="courseSemester" placeholder="Spring/Summer/Fall/Winter and year.  Ex. Fall 2018" title="When is this course taking place?"required>
+		</div>
+		<!-- Course type -->
+		<div class="field" title="A client course is looking to outsource specific tasks. A supplier course is looking for projects to team up with.">
+			<label for="courseType">Course type</label>
+			<select class="ui fluid search dropdown" name="courseType" id="courseType"  required>
+				<option value=""></option>
+				<option value="1">Client</option>
+				<option value="2">Supplier</option>
+			</select>
+		</div>
+		<!-- Course schedule -->
+		<div class="ui grid courseSchedule">
+			<div class="eight wide column field">
+				<label for="courseSchedule">Course schedule</label>
+				<select class="ui fluid search dropdown" name="courseType" id="courseSchedule" multiple required>
```

Comment:
Change name attribute from "courseType" to "courseSchedule"

Reply:
Add "[]" at the end of the name attribute, to ensure this input's contents are sent as a list to back-end

Reply:
Done! Thank you 😄